Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM gcc:latest as build
+
+WORKDIR /app
+
+RUN set -ex; \
+    apt-get update; \
+    apt-get install -y cmake
+
+COPY . /app/
+
+RUN sed -i 's/MARIADB_CONNECTOR_COMPILE_JOBS__=1/MARIADB_CONNECTOR_COMPILE_JOBS__=2/' /app/CMakeVariables.txt 
+
+RUN mkdir build
+RUN cd build && \
+	cmake .. && \
+	make -j2
```

Comment:
Can this not be replaced with `./build.sh -j2`?  That file does the same work as is done here